test/Maybe/Maybe.js

```diff
@@ -65,6 +67,7 @@ var RightArb = function(arb) {
 var Compose = function(F, G) {
   var _Compose = function _Compose(x) {
     return {
+      constructor: _Compose,
```

Comment:
In order to test both sides of `hasMethod('of')(x) || hasMethod('of')(x.constructor)` I defined `of` on "instances" for Identity and on the "constructor" for Compose.